libs/cardano-ledger-binary/cardano-ledger-binary.cabal

```
@@ -106,7 +106,7 @@ library testlib
         cardano-ledger-binary,
         cardano-prelude-test,
         cardano-strict-containers,
-        cardano-slotting,
+        cardano-slotting:{cardano-slotting, testlib},
```

We need to add a lower bound for the version that contains those instances
```suggestion
        cardano-slotting:{cardano-slotting, testlib} >= 0.1.2,
```

---

Done!